### DISCUSSION TITLE
Recommended way to export resources from a plugin (RC5 and above)

### DISCUSSION THREAD

**Discussion by DamianGlowala** [nuxt/nuxt#17245] [Questions]
Is it the recommended way to write a plugin which imports other resources (starting from the RC5 release)?

```
import * as LottiePlayer from "@lottiefiles/lottie-player";

export { LottiePlayer };

export default defineNuxtPlugin(() => {});
```

**Comment by pi0** [comment]
Hi @DamianGlowala. I would recommend to migrate to `composables/` directory for this situations. They are tree-shakable, auto imported and only added to the chunks when actually needed while plugins are globally added to the entrypoint chunk. 

BTW what has been changed since RC.5?

**Reply by danielroe** [reply]
I think the warning to wrap plugins in `defineNuxtPlugin`.

**Reply by pi0** [reply]
Oh then if that's the case (plugin file has no function export), definitely go for `composables/` 🚀 (unless the purpose of plugin is global side-effect which still I would keep it in plugin function with a flag to only run once)
